ci/container/container-run.sh

```diff
@@ -160,22 +160,33 @@ mkdir -p ~/.{aws,ssh,cache,local/share/fish} && touch ~/.{zsh,bash}_history
 PODMAN_RUN_USR_ARGS=()
 if [ -f "$HOME/.container-run.conf" ]; then
     # conf file with user's custom PODMAN_RUN_USR_ARGS
-    echo "Sourcing user's ~/.container-run.conf"
+    echo "Sourcing user's ~/.container-run.conf" >&2
     source "$HOME/.container-run.conf"
 fi
 
+# Omit -t if not a tty.
+# Also shut up logging, because podman will by default loge
+# every byte of standard output to the journal, and that
+# destroys the journal + wastes enormous amounts of CPU.
+# I witnessed journald and syslog peg 2 cores of my devenv
+# when running a simple cat /path/to/file.
+if tty >/dev/null 2>&1; then
+    tty=" -t --log-driver=none "
```

Comment:
nit: use an array and quote the variable below ( `"${tty[@]}"`)

Reply:
No.  This is meant to be interpreted unquoted.  This is not an accident but a deliberate goal of the code.